html-aam/index.html

```diff
@@ -13368,7 +13368,7 @@ <h4 id="att-popover">`popover`</h4>
                   <li>when the element has a `popovertarget=hide` attribute value,</li>
```

Comment:
```suggestion
                  <li>when the element has a `popovertargetaction=hide` attribute value,</li>
```

I know this isn’t related to this PR, but should this be `popovertargetaction`?

Reply:
yes!  thank you!